tinkerbell/stack/templates/kubevip.yaml

```diff
@@ -21,12 +21,12 @@ spec:
         env:
         - name: vip_arp
           value: "true"
-        - name: vip_cidr
```

Comment:
What does this default to?

Reply:
from the kubevip docs:
`vip_cidr | Defaults "32" | Used when advertising BGP addresses (typically as x.x.x.x/32)`